rust/compaction/src/datafusion/ageoff_udf.rs

```diff
@@ -39,7 +39,7 @@ use std::{
 /// lower than the given threshold, it will be filtered out.
 #[derive(Debug)]
 pub struct AgeOff {
-    /// Threshold value (seconds since UNIX epoch)
+    /// Threshold value (milliseconds since UNIX epoch)
     threshold: i64,
```

Comment:
There's corresponding documentation aggregate_udf.rs that also needs updating. It still says seconds there.